cirq/_version.py

```diff
@@ -12,13 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Define version number here, read it from setup.py automatically, and warn users that
-the latest version of cirq uses python 3.6+"""
+"""Define version number here, read it from setup.py automatically,
+and warn users that the latest version of cirq uses python 3.6+"""
 
 import sys
 
 if sys.version_info < (3, 6, 0):
-    raise SystemError(
+    # TODO: Find the best way to cover this. Maybe mock sys.version_info
```

Comment:
This does not need to be covered. Just mark it `# coverage: ignore`.

Reply:
Done.